lib/transaction/index.js

```diff
@@ -162,7 +163,7 @@ transaction.prototype = object.create(EventEmitter.prototype);
 
 /**
  * @param {object} data
- * @param {otp|push|sms} data.method
+ * @param {otp|push|sms|recovery-code} data.method
```

Comment:
Is method going to include recovery code? doesnt that cause the issue we talked about?